pkg/proxy/iptables/proxier.go

```diff
@@ -294,10 +294,14 @@ func NewProxier(ipt utiliptables.Interface,
 		ipFamily = v1.IPv6Protocol
 	}
 
-	var incorrectAddresses []string
-	nodePortAddresses, incorrectAddresses = utilproxy.FilterIncorrectCIDRVersion(nodePortAddresses, ipFamily)
-	if len(incorrectAddresses) > 0 {
-		klog.Warningf("NodePortAddresses of wrong family; %s", incorrectAddresses)
+	isCIDR := true
+	ipFamilyIPMap := utilproxy.MapIPsToIPFamily(nodePortAddresses, isCIDR)
```

Comment:
I think I'd rather see `MapIPsByIPFamily` and `MapCIDRsByIPFamily` as distinct functions, rather than a bool arg

Reply:
Sounds good. Will make this change

Reply:
Implemented 2 separate functions MapIPsByIPFamily and MapCIDRsByIPFamily. Added go tests for both.